routers/organizationRouter.go

```diff
@@ -277,59 +256,20 @@ func (handler *organizationHandler) RemoveMemberFromOrganization(writer http.Res
 	}
 
 	token := request.Context().Value(jwtmiddleware.ContextKey{}).(*validator.ValidatedClaims)
+	tokenCustomClaims := token.CustomClaims.(*auth.CustomClaims)
 	userId := token.RegisteredClaims.Subject
 	removeUsersPerm := fmt.Sprintf("org%s:remove_members", organizationId)
-	canRemoveUsers, err := auth.HasPermission(userId, removeUsersPerm)
-	if err != nil {
-		http.Error(writer, fmt.Sprintf("Failed to get user with id %s permissions: %s", userId, err.Error()), http.StatusInternalServerError)
-		return
-	}
+	canRemoveUsers := tokenCustomClaims.HasPermission(removeUsersPerm)
 	if !canRemoveUsers {
 		http.Error(writer, fmt.Sprintf("User with id %s does not have permission to remove users from organization with id: %s", userId, organizationId), http.StatusForbidden)
 		return
 	}
 
-	err = handler.controller.RemoveMember(memberId, organizationId)
+	err := handler.controller.RemoveMember(memberId, organizationId)
 	if err != nil {
 		http.Error(writer, fmt.Sprintf("Failed to remove user with id %s from org with id %s: %s", memberId, organizationId, err.Error()), http.StatusInternalServerError)
 		return
 	}
 	writer.Header().Set("Content-Type", "application/json")
 	writer.WriteHeader(http.StatusNoContent)
 }
-
-func (handler *organizationHandler) GetAllBoards(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Same comment as above (while i know your method is more effective I am just double checking that this was intentionally removed rather than a github issue)

Reply:
Yeah this was me moving it to the other router